src/quartics/ferrari_qs_functor.py

```diff
@@ -0,0 +1,216 @@
+"""
```

Comment:
I would change `src/quartics/` to `src/solvers`. Then change this file to `ferrari.py`